config/locales/es/es.yml

```diff
@@ -389,6 +389,7 @@ es:
 
   project_group:
     add: "Translate Me"
+    edit: "Translate Me"
```

Comment:
These things are not necessary. Our translation system detects missing strings. Thanks though!